package.json

```diff
@@ -7,7 +7,6 @@
   "devDependencies": {
     "@fortawesome/fontawesome-free": "^5.15.1",
     "bootstrap": "^4.5.3",
-    "bootstrap-datetime-picker": "^2.4.4",
```

Comment:
Need to update package-lock.json as well - running `npm install` should remove it from there.